app/Jobs/SendSignupToPhoenix.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Signup;
+use Rogue\Services\Phoenix;
+use Rogue\Services\Registrar;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendSignupToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue, SerializesModels;
+
+    /*
+     * Signup instance.
+     *
+     * @var Rogue\Models\Signup
+     */
+    protected $signup;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Signup $signup)
+    {
+        $this->signup = $signup;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle(Phoenix $phoenix, Registrar $registrar)
```

Comment:
@sbsmith86 is it cool if I handle updates not used by the signup endpoint in separate PRs? This is pretty big and I don't want it to get too scattered focus wise or be too much to test all at once. If that's cool I'll make issues for the other changes so they don't get forgotten!